data/json/items/ammo/8x40mm.json

```diff
@@ -0,0 +1,104 @@
+[
+  {
+    "id": "8mm_bootleg",
+    "copy-from": "8mm_jhp",
+    "type": "AMMO",
+    "name": "bootleg 8x40mm JHP",
+    "name_plural": "bootleg 8x40mm JHP",
+    "description": "Bootleg duplicates of Rivtech 8x40mm caseless rounds.  Being caseless rounds, these cannot be disassembled or reloaded.",
+    "proportional": {
+      "price": 0.7,
+      "damage": 0.9,
+      "dispersion": 1.1
+    },
+    "extend": { "effects": [ "RECYCLED" ] },
+    "delete": { "effects": [ "NEVER_MISFIRES" ] }
+  },
+  {
+    "id": "8mm_caseless",
+    "type": "AMMO",
+    "name": "8x40m",
+    "name_plural": "8x40m",
+    "//": "Cased ammo tends to be roughly $1/shot, more or less.  Rivtech ammo, being New and Proprietary and Expensive, $2-2.50 or so.",
+    "description": "8x40mm caseless rounds.  Proprietary ammunition for Rivtech firearms.  Being caseless rounds, these cannot be disassembled or reloaded.",
+    "weight": 12,
+    "volume": 1,
+    "price": 12500,
+    "material": [ "plastic", "powder" ],
+    "symbol": "=",
+    "color": "pink",
+    "count": 40,
+    "stack_size": 40,
+    "ammo_type": "8x40mm",
+    "range": 42,
+    "damage": 42,
+    "pierce": 18,
+    "dispersion": 75,
+    "recoil": 400,
+    "effects": [ "NEVER_MISFIRES" ]
+  },
+  {
+    "id": "8mm_civilian",
+    "copy-from": "8mm_caseless",
+    "type": "AMMO",
+    "name": "8x40m sporting",
+    "name_plural": "8x40m sporting",
+    "description": "8x40mm caseless rounds, light sporting load sold for civilian use.  Proprietary ammunition for Rivtech firearms.  Being caseless rounds, these cannot be disassembled or reloaded.",
+    "proportional": {
+      "price": 0.8,
+      "damage": 0.5,
+      "dispersion": 1.2,
+      "recoil": 0.5
+    },
+    "delete": { "effects": [ "NEVER_MISFIRES" ] }
+  },
+  {
+    "id": "8mm_fmj",
+    "copy-from": "8mm_caseless",
+    "type": "AMMO",
+    "name": "8x40mm FMJ",
+    "name_plural": "8x40mm FMJ",
+    "description": "8x40mm caseless rounds, full metal jacket.  Military grade ammunition for Rivtech firearms.  Being caseless rounds, these cannot be disassembled or reloaded.",
+    "relative": {
+      "damage": -5,
+      "pierce": 10
+    }
+  },
+  {
+    "id": "8mm_hvp",
```

Comment:
> This is expected and intended, obscure ammo types are something to be managed, not remived.

The problems start when you can't even fire a single burst before reloading despite having enough ammo in total and magazine capacity for 3.
IRL this would probably result in mixing them in one mag, in the game it means having to reload each time ammo of a single rare type runs out.